src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -2328,6 +2328,18 @@ void Convert::pedalFromMEI(engraving::Pedal* pedal, const libmei::Pedal& meiPeda
         pedal->setEndHookType(engraving::HookType::HOOK_90);
     }
 
+    // @func
+    if (meiPedal.GetFunc() == "soft") {
+        if (pedal->beginText() != "") {
+            pedal->setBeginText(u"una corda");
+        }
+    } else if (meiPedal.GetFunc() == "sostenuto") {
+        if (pedal->beginText() != "") {
+            pedal->setBeginText(u"<sym>keyboardPedalSost</sym>");
+            pedal->setContinueText(u"<sym>(keyboardPedalSost)</sym>");
```

Comment:
Shoulld probably be ` pedal->setContinueText(u"(<sym>keyboardPedalSost</sym>)");`